.npmrc

```diff
@@ -1 +1,3 @@
 resolution-mode=highest
+hoist=true
+hoist-workspace-packages=true
```

Comment:
This configuration would likely indicate problems with peer dependencies / dependencies of packages. 
It used to be the trick in yarn to make monorepos work.

We should make the package resolutions stricter rather than relaxing it.
e.g.

```
# resolution-mode is here because of old pnpm problem in pre-8.0. This setting is now the default
resolution-mode=highest
auto-install-peers=false
resolve-peers-from-workspace-root=false
```

Reply:
Reverted this; but I'm still having the issue in #2864